openpathsampling/movechange.py

```diff
@@ -365,6 +365,26 @@ def _get_results(self):
         return []
 
 
+class RejectedNaNSampleMoveChange(RejectedSampleMoveChange):
+    """
+    Represents an rejected SamplePMC because of occurance of NaN
+
+    This will return no samples also as its result, hence it is
+    rejected.
+    """
+    pass
+
+
+class RejectedMaxLengthSampleMoveChange(RejectedSampleMoveChange):
+    """
+    Represents an rejected SamplePMC because of occurance of NaN
```

Comment:
please correct the docstring here: this one is max length, not NaN.

Question about both of these: they return empty `result`, but do they return empty `trials`? I think that it is worth trying to return the (invalid) trial if possible.